pkg/transformers/refvars.go

```diff
@@ -39,6 +39,17 @@ func (rv *refvarTransformer) replaceVars(in interface{}) (interface{}, error) {
 			xs = append(xs, expansion.Expand(a.(string), rv.mappingFunc))
 		}
 		return xs, nil
+	case map[string]interface{}:
+		inMap := in.(map[string]interface{})
+		xs := make(map[string]interface{}, len(inMap))
+		for k, v := range inMap {
+			s, ok := v.(string)
+			if !ok {
+				return nil, fmt.Errorf("%#v is expected to be %T", v, s)
+			}
+			xs[k] = expansion.Expand(s, rv.mappingFunc)
+		}
+		return xs, nil
```

Comment:
i realize that `refvars_test.go` isn't there, but it's never too late to start coverage.

The format would be to make a ResMap with just the object of interest in it, make a refVarTransformer, call Transform, etc.  (e.g. prefixsuffixname_test)

The high level target test is appreciated, but we'd like to get lower level coverage too, to capture oddball stuff like switching on an interface type in a recursion.

Reply:
I realized there were no tests too ;)

I added a test case and I paved the way to add more easily!

> The high level target test is appreciated, but we'd like to get lower level coverage too,

agree, high level target tests  gives me more confidence in that all parts are glued and working together, and true, lower unit tests helps to better catch oddball